### PR TITLE
Re-Enable it

### PR DESCRIPTION
**Pull request by jcantrill**
This PR re-enables integration tests

### CODE REVIEW

File: src/test/java/io/fabric8/elasticsearch/plugin/HttpsProxyClientCertAuthenticatorIntegrationTest.java [hunk 1]
```diff
@@ -1,73 +0,0 @@
...
-import org.apache.http.client.fluent.Request;
-import org.apache.http.client.fluent.Response;
-import org.apache.http.conn.ssl.TrustSelfSignedStrategy;
-import org.apache.http.impl.client.CloseableHttpClient;
-import org.apache.http.impl.client.HttpClients;
-import org.apache.http.ssl.SSLContextBuilder;
-import org.junit.Before;
-import org.junit.Test;
-
-public class HttpsProxyClientCertAuthenticatorIntegrationTest {
```
Comment (ewolinetz) [inline src/test/java/io/fabric8/elasticsearch/plugin/HttpsProxyClientCertAuthenticatorIntegrationTest.java]: I forget why we need to remove this one...

**Comment by ewolinetz** [comment]
one question otherwise LGTM

**Comment by jcantrill** [comment]
[merge]

**Pull request merged**